misc-tools/dj_make_chroot.in

```diff
@@ -227,34 +236,24 @@ mkdir -p "$CHROOTDIR"
 cd "$CHROOTDIR"
 CHROOTDIR="$PWD"
 
-if [ ! -x /usr/sbin/debootstrap ]; then
-
-    echo "This script will install debootstrap on your system."
-    if [ ! "$FORCEYES" ]; then
-        printf "Continue? (y/N) "
-        read -r yn
-        if [ "$yn" != "y" ] && [ "$yn" != "Y" ]; then
-            exit 1;
-        fi
-    fi
-
-    if [ -f /etc/debian_version ]; then
-
-        cd /
-        apt-get install debootstrap
-
-    else
-        mkdir "$CHROOTDIR/debootstrap"
-        cd "$CHROOTDIR/debootstrap"
-
-        wget "$DEBMIRROR/pool/main/d/debootstrap/${DEBOOTDEB}"
-
+if [ "$FORCEDOWNLOAD" = 0 ] && command -v debootstrap >/dev/null; then
+    BOOTSTRAP_COMMAND=$(command -v debootstrap)
+    echo "Using debootstrap from $BOOTSTRAP_COMMAND"
+elif [ "$FORCEDOWNLOAD" = 0 ] && [ -f /etc/debian_version ]; then
+    confirmation "Do you want to install debootstrap using apt-get?" "debootstrap is required, exiting."
+    apt-get install -y debootstrap
```

Comment:
Can we change this to "using APT" in the question and the command to `apt install` while we're at it?

Reply:
I've mixed feelings about that. It seems that `apt-get` is more suitable for scripts than `apt` ([source](https://askubuntu.com/questions/990823/apt-gives-unstable-cli-interface-warning)). That refers mainly to the output format of `apt` which may not be stable. In this particular instance it may not be an issue since we are not parsing its output and I highly doubt that they will ever change the flags of `apt`.

What do you think?

Reply:
Alright, it seems OK to remain with apt-get, although on the other hand we don't parse the output indeed.